### ISSUE TITLE
cyr_virusscan feature-request: moving infected messages

### ISSUE THREAD

**Issue by sueba** [cyrusimap/cyrus-imapd#1840]
Wouldn't it be a good idea to add a switch for  moving infected messages to another folder? That would allow for some sort of quarantine solution in case of detection.

**Comment by elliefm** [comment]
I can see two ways this could go:

* move the messages to a mailbox in the same user's account (so if user.anne has an infected message, the message in question is moved to, say, user.anne.quarantine)
* move the messages to a specific mailbox (so if any user has an infected message, the message in question is moved to, say, user.quarantine)

I think the second option seems most useful practically (stops user from accidentally infecting themselves, message is still available if it turns out to have been a false positive).  The first option seems potentially more flexible but also more confusing -- for admins and users alike.

**Comment by hagedose** [comment]
I agree. I think ideally each account would get an additional mailbox there, so that for your example the message would be moved to user.quarantine.anne. You could run ipurge on those mailboxes to remove messages after some time. Users could be given access to them by some mechnanism  that sets a read ACL for a limited time.

**Comment by rjbs** [comment]
:robot: I'm closing this issue by automation.

This issue has been sitting untouched for over eight years.  We're trying to get a good handle on our open issues and pull requests, so that we can reliably use them as a backlog of things we hope to work on.  If you are confident that this issue should be reconsidered, reopen it and explain why.

Remember, though, that there are only so many developers working on Cyrus, and we'd like to focus on the most important and/or low-hanging work.

Before you reopen this issue, please check that this issue is still present in the latest release of Cyrus, or the main development branch. An issue is considerably more likely to get resolved if you can provide a pull request which reproduces the issue in a Cassandane test.
